manifests/init.pp

```diff
@@ -163,6 +163,10 @@
 # @param group
 #   Primary group for the Candlepin user
 #
+# @param disable_fips
+#   Disable FIPS within the Java environment for Tomcat explicitly.
+#   When set to false no flag is added and system configuration is used.
```

Comment:
```suggestion
#   When set to false no flag is added and system configuration is used.
#   When set to false and system is configured in FIPS mode, a Candlepin build that supports FIPS is required.
```

Reply:
It wasn't immediately clear to me. The original wasn't either. How about
```suggestion
#   When set to false, no flag is added. Then on FIPS enabled systems, a Candlepin build that supports FIPS is required.
```

Reply:
works for me.